### PR TITLE
blog pic

### PR DESCRIPTION
**Pull request by prank15062002**
pic2
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -203,4 +203,9 @@
   img: DSC_3288.jpg
   author_name: Aman Singh
   
+- url: https://openblogsource.wordpress.com/2017/12/28/my-journey-open-source/
+  title: my-journey- open-source
+  img: opensourceblog.png
```
Comment (nikit19) [inline _data/blogs.yml]: Change the name of the image to the one that is of 240 x 240 pixels ie pic2.jpg

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please delete the other image which is not of the specified dimensions.

File: _data/blogs.yml (hunk 1 above)
Comment (prank15062002) [reply _data/blogs.yml]: i have added another pull request  and have deleted this image

**Comment by nikit19** [comment]
You have uploaded two images. Please delete the one you have uploaded in the img folder.
Rest looks good.

**Comment by prank15062002** [comment]
i have deleted the unused image

**Comment by prank15062002** [comment]
how much time will it take for my work to be reviewed @nikit19

**Comment by nikit19** [comment]
You have uploaded two images. Please delete the one you have uploaded in the img folder.
And also resolve conflicts

File: _data/blogs.yml [hunk 2]
```diff
@@ -203,4 +203,9 @@
   img: DSC_3288.jpg
   author_name: Aman Singh
   
+- url: https://openblogsource.wordpress.com/2017/12/28/my-journey-open-source/
+  title: my-journey- open-source
+  img: pic2.jpg
+  author_name: Pranjal Rastogi
+
```
Comment (nzec) [inline _data/blogs.yml]: Don't put 2 spaces at the bottom.

**Comment by nzec** [comment]
@prank15062002 You need to delete the unused image and then commit and push. Also do the changes I suggested above. Also resolve the merge conflicts!

Other than that, it looks good. And nice blog by the way!

**Comment by anshumanv** [comment]
Please remove image `img/pic2.jpg` and resolve conflicts.

**Comment by prank15062002** [comment]
I am unable to resolve conflict , Therefore I am closing pull request and opening a new one

**Pull request closed without merging**